tests/helper/helper_cluster.go

```diff
@@ -13,36 +13,36 @@ import (
 	"github.com/onsi/gomega/gexec"
 )
 
-type OcRunner struct {
-	// path to oc binary
+type ClusterRunner struct {
+	// path to cluster binary
 	path string
 }
 
-// NewOcRunner initializes new OcRunner
-func NewOcRunner(ocPath string) OcRunner {
-	return OcRunner{
-		path: ocPath,
+// NewClusterRunner initializes new ClusterRunner
+func NewClusterRunner(clusterPath string) ClusterRunner {
+	return ClusterRunner{
+		path: clusterPath,
 	}
 }
 
-// Run oc with given arguments
-func (oc *OcRunner) Run(cmd string) *gexec.Session {
+// Run cluster with given arguments
+func (cluster *ClusterRunner) Run(cmd string) *gexec.Session {
 	session := CmdRunner(cmd)
 	Eventually(session).Should(gexec.Exit(0))
 	return session
 }
 
 // SwitchProject switch to the project
-func (oc *OcRunner) SwitchProject(projectName string) {
+func (cluster *ClusterRunner) SwitchProject(projectName string) {
```

Comment:
replace with something like this ```func (oc *CliRunner) SwitchProject(projectName string) {``` because this is an OpenShift specific function and it won't be supported for ```kubectl``` command. Find out all those type of function and update accordingly. @prietyc123

Reply:
Yes, we can replace ```cluster``` with ```cli``` but if we try to replace specifically go-lint will throw error as ```receiver name oc should be consistent with previous receiver name cli for CliRunner go-lint```